src/WCSimDetectorConfigs.cc

```diff
@@ -20,7 +20,10 @@
 
 void WCSimDetectorConstruction::SetSuperKGeometry()
 {
-  WCSimPMTObject * PMT = CreatePMTObject("PMT20inch", "glassFaceWCPMT");
+  WCDetectorName = "SuperK";
```

Comment:
Not required, but this might be a nice thing to store as a branch in the geometry tree. That way you can quickly determine what geometry was used without needing to look at the PMT arrangement or rely on the filename. Could be especially handy when evaluating multiple types of geometry.